style.css

```diff
@@ -0,0 +1,109 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap');
+
+body {
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  margin-left: 40px;
+  margin-top: 40px;
+  background: #fff;
```

Comment:
- [ ] Kindly 😃 try to give the known style for the body and all elements, so please at the body here try to remove all margins from the body to let the body start from the beginning of the edges 😊 , so try to put these properties for the style it is a general style for all elements :
```code
*{
padding: 0;
margin: 0;
box-sizing: border-box;
}
```
- [ ] Also Please 😊try to remove the width of the body you make the mobile version first after that you can make the desktop version using the media query instead of the width directly 👌

Reply:
Done